components/zenler/actions/enroll-user/enroll-user.mjs

```diff
@@ -0,0 +1,48 @@
+import zenler from "../../zenler.app.mjs";
+
+export default {
+  key: "zenler-enroll-user",
+  name: "Enroll User",
+  description: "Enrolls a user to a course. [See the docs here](https://www.newzenler.com/api/documentation/public/api-doc.html#faaadc39-5702-bb2c-640f-c76a47c26f81)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    zenler,
+    userId: {
+      propDefinition: [
+        zenler,
+        "userId",
+      ],
+    },
+    courseId: {
+      propDefinition: [
+        zenler,
+        "courseId",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const {
+      userId,
+      courseId,
+    } = this;
+
+    const response = await this.zenler.enrollUser({
+      userId,
+      data: {
+        course_id: courseId,
+      },
+    });
+
+    if (typeof(response) === "string") {
+      console.log(response);
+      throw new Error("Response error");
```

Comment:
Is it possible to specify the error in the `throw` message?

Reply:
Actually when this happens the response comes in with HTML format, and it doesn't say much. It just says something like "Something went wrong" that's why I've just put `Response error`

Reply:
Got it! tks!